pkg/users/parsers.js

```diff
@@ -77,3 +77,19 @@ function parse_group_content(content) {
 export const etc_group_syntax = {
     parse: parse_group_content
 };
+
+function parse_shells_content(content) {
+    content = (content || "").trim();
+    if (!content) {
+        console.warn("Couldn't read /etc/shells");
+        return [];
+    }
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230220-124215-f7f3850d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)